apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -68,12 +190,38 @@
 
   li {
     display: inline-block;
-    width: 100px;
-    margin: 0 10px;
+    width: 49%;
+    height: 0;
+    margin: 0;
```

Comment:
Properties should be ordered display, width, height, padding-bottom, margin, margin-bottom, cursor, background-color, background-repeat, background-position, background-size

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)